charts/core/values.yaml

```diff
@@ -254,7 +254,7 @@ apiGateway:
   additionalHosts: []
   # -- The TLS configuration for api-gateway
   tls:
-    enabled: false
+    enabled: true
```

Comment:
Don't change this. Let's keep the default be false here.
We'll override it in Terraform when deploying to cluster.

Reply:
updated. Thanks